### ISSUE TITLE
[Enhancement] Ability to turn on/off display

### ISSUE THREAD

**Issue by TheOnlyJonto** [kiteretro/Circuit-Sword#42]
I would like to be able to turn the display off and on easily in order to save battery without shutting the system down.

**Comment by jamesalmeida** [comment]
Maybe turning the brightness all the way down and then going down 1 more can make the screen turn off? Not sure if that's possible but would be a cool way to incorporate the feature.

**Comment by TheOnlyJonto** [comment]
Problem is adjusting the brightness from max to min takes forever. I'd rather see this implemented as its own command so I can turn it on and off as quickly as possible.

**Comment by jamesalmeida** [comment]
Then maybe holding mode+Left for 3 seconds can skip lowering brightness and just turn off screen. 

Or maybe Mode + Right then Left 

Any other ideas or suggestions on which button combo could be used that’s available?

**Comment by TheOnlyJonto** [comment]
Well, as far as I know, left and right aren't used for any mode button combos so it could just be a simple mode+left/right to turn the display on and off. No need to hold it down.

**Comment by jamesalmeida** [comment]
I believe by default `MODE+LEFT` lowers the screen brightness and `MODE+RIGHT` increases the screen brightness.

**Comment by TheOnlyJonto** [comment]
No up and down control screen brightness.

**Comment by jamesalmeida** [comment]
Ah then Left and Right controls Volume.

**Comment by jamesalmeida** [comment]
Maybe because my board has HDMI instead of the volume wheel it needs the MODE control otherwise I can't set my volume. The original board had a volume wheel but the newer Circuit Sword has 2 different back boards. 1 with HDMI and the other has a volume wheel instead of the HDMI out. 

He mentions the 2 backboards on the first 2 questions on the wiki page: https://github.com/kiteretro/Circuit-Sword/wiki/FAQ

And you can see the HDMI backboard in the picture in the first post here:
https://sudomod.com/forum/viewtopic.php?f=38&t=5212

**Comment by TheOnlyJonto** [comment]
Ah. I do not have the circuit sword. I guess the rear buttons could be used for this function then.

**Comment by jamesalmeida** [comment]
Good point. I don’t think the back buttons current do anything with the MODE button.
